src/pages/ServerRoleSummary/ServerGroupsTab.js

```diff
@@ -59,6 +60,25 @@ class ServerGroupsTab extends Component {
     }
   }
 
+  getServerGroupsInUse = (internalModel) => {
+    let serverGroupList =
+      internalModel['internal']['servers']?.filter(server => !isEmpty(server['ardana_ansible_host']))
+        .map(server => server['server-group-list']);
+    // flat and remove the duplicates
+    serverGroupList =[...new Set([].concat(...serverGroupList))];
```

Comment:
Ditto. Needs a comment about serverGroupList being a list of lists or whatever it is.

Reply:
@GarySmith  ok will add comments